src/main/java/com/microsoft/sqlserver/jdbc/ParameterMetaDataCache.java

```diff
@@ -8,19 +8,22 @@
 import java.util.AbstractMap;
 import java.util.ArrayList;
 import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
+
+import mssql.googlecode.concurrentlinkedhashmap.ConcurrentLinkedHashMap;
+import mssql.googlecode.concurrentlinkedhashmap.ConcurrentLinkedHashMap.Builder;
 
 
 /**
  * Implements a cache for query metadata returned from sp_describe_parameter_encryption calls. Adding, removing, and
  * reading from the cache is handled here, with the location of the cache being in the EnclaveSession.
  * 
  */
-class ParameterMetaDataCache {
-
-    static final int CACHE_SIZE = 2000; // Size of the cache in number of entries
-    static final int CACHE_TRIM_THRESHOLD = 300; // Threshold above which to trim the cache
+public class ParameterMetaDataCache {
 
+    static int CACHE_SIZE = 2000; // Size of the cache in number of entries
+    static int MAX_WEIGHTED_CAPACITY = 2300; // Size of cache + threshold, above which we trim.
+    static int CACHE_HITS = 0; // Record of the number of cache hits, mainly for testing

```

Comment:
this gets incremented but is never used in the driver anywhere

Reply:
I used it in testing. I used it to monitor whether the cache is working correctly, as a successful `get` will increment `CACHE_HITS`

Reply:
Ok this was dumb. Removed and resolved.